cmd/xmidt-agent/xmidt_agent.yaml

```diff
@@ -1,33 +1,6 @@
 # SPDX-FileCopyrightText: 2023 Comcast Cable Communications Management, LLC
```

Comment:
I figured we'll want to trim everything that is not required for `cmd/xmidt-agent/main_test.go`